pkgs/development/libraries/fribidi/default.nix

```diff
@@ -6,9 +6,10 @@
 , pkg-config
 , fixDarwinDylibNames
 , python3
+, testers
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: rec {
```

Comment:
When using the `mkDerivation (finalAttrs: { })` form, it's important to remove `rec` from the attrset. Otherwise it defeats the purpose of `finalAttrs`, and becomes confusing.

Wherever a derivation attribute is needed, look it up from `finalAttrs`. E.g. `finalAttrs.version`.